data/mods/Aftershock/items/armor/winter_suits.json

```diff
@@ -0,0 +1,126 @@
+[
+  {
+    "id": "afs_magellan_suit",
+    "type": "TOOL_ARMOR",
+    "category": "armor",
+    "name": { "str": "Magellan exosuit" },
+    "description": "A high-quality, civilian grade EVA suit often employed by well-established frontier research and exploration associations.  Designed to support the exploration of challenging terrain, it offers respectable protection against common environmental hazards like extreme temperatures, inhospitable atmospheres, and light radiation.  It leaves arms and hands relatively unencumbered to aid the manipulation of scientific instruments.\n\nAn integral battery allows the suit to operate for up to 36 hours, but complicates field recharging",
```

Comment:
```suggestion
    "description": "A high-quality, civilian grade EVA suit often employed by well-established frontier research and exploration associations.  Designed to support the exploration of challenging terrain, it offers respectable protection against common environmental hazards like extreme temperatures, inhospitable atmospheres, and light radiation.  It leaves arms and hands relatively unencumbered to aid the manipulation of scientific instruments.\n\nAn integral battery allows the suit to operate for up to 36 hours, but complicates field recharging.",
```

Reply:
Good spotting! (I'm not sure why I miss those so much - perhaps the comma at the end?)